docs/sources/setup-grafana/configure-security/planning-iam-strategy.md

```diff
@@ -0,0 +1,197 @@
+
+## Benefits of integrating with an IAM provider
+
+Integrating with an IAM provider provides the following benefits:
+
+- **User management**: By providing Grafana access to your current user management system, you eliminate the overhead of replicating user information and instead have centralized user management for users' roles and permissions to Grafana resources.
+
+- **Security**: many IAM solutions provide advanced security features such as multi-factor authentication, RBCA, and audit trails, which can help to improve the security of your Grafana installation.
+
+- **SSO**: Properly setting up Grafana with your current IAM solution enables users to access Grafana with the same credentials they use for other applications.
+
+- **Scalability**: User additions and updates in your user database are immediately reflected in Grafana.
+
+In order to plan an integration with Grafana, assess your organization's current needs, requirements and any existing IAM solutions being used. This includes thinking about how roles and permissions will be mapped to users in Grafana, and how users can be grouped together to access shared resources.
+
+## Internal vs external users
+
+As a first step, determine how you want to manage users who will access Grafana.
+
+Do you already use an identity provider to manage users? If so, Grafana might be able to integrate with your identity provider through one of our IdP integrations.
+Refer to [Configure authentication documentation]({{< relref "./configure-authentication" >}}) for the list of currently supported providers.
+
+If you are not interested in setting up an external identity provider, but still want to limit access to your Grafana instance, you should consider using Grafana's basic authentication.
+
+Finally, if you want your Grafana instance to be accessible to everyone, you can enable anonymous access to Grafana.
+For information, refer to the [anonymous authentication documentation]({{< relref "./configure-security/configure-authentication#anonymous-authentication" >}}).
+
+## Ways to organize users
+
+Organize users in subgroups that are sensible to the organization. These are some examples:
+
+- **Security**: Different groups of users or customers should only have access to their intended resources.
+- **Simplicity**: Reduce the scope of dashboards and resources available.
+- **Cost attribution**: Track and bill costs to individual customers, departments, or divisions.
+- **Customization**: Each group of users could have a personalized experience like different dashboards or theme colours.
+
+### Users in Grafana teams
+
+You can organize users into [teams] and assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team.
```

Comment:
```suggestion
You can organize users into [teams] and assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team of those users and assign dashboard permissions to the team.
```